xarray/core/options.py

```diff
@@ -269,3 +271,56 @@ def __enter__(self):
 
     def __exit__(self, type, value, traceback):
         self._apply_update(self.old)
+
+
+def get_options():
+    """Returns options for xarray.
+
+    Currently supported options:
+
+    - ``display_width``: maximum display width for ``repr`` on xarray objects.
```

Comment:
A new format for the parameters was just merged, #5702. Could you copy that one instead?

Reply:
Thanks for pointing that out, I've just copied it now.